### PR TITLE
50 2 frontend fetches mapping

### PR DESCRIPTION
**Pull request by bravint**

### CODE REVIEW

**Review by nathanSamosa — APPROVED** [review]

File: src/client/components/Season.jsx [hunk 1]
```diff
@@ -1,47 +1,103 @@
...
+    useEffect(() => {
+        const id = 2;
+
+        const filteredSeason = competitions[0].seasons.filter((season) => season.id === id);
+
+        SetSelectedSeason(filteredSeason);
+    }, []);
+
+    useEffect(() => {
+        if (!selectedSeason.length) {
```
Comment (polarbear23) [inline src/client/components/Season.jsx:30]: any reason why you dont one line this?

**Review by molloyconor511 — APPROVED** [review]
You have my approval good sir!

**Review by polarbear23 — APPROVED** [review]

File: src/client/components/Season.jsx (hunk 1 above)
Comment (nathanSamosa) [reply src/client/components/Season.jsx:30]: Its just to upset you

Comment (bravint) [reply src/client/components/Season.jsx:30]: I used to, but Dr. Huckle said no. Then Dr. King said no. Then Alexa said no. I can change if you guys are one-lining the if statements though.

**Comment by bravint** [comment]
change if statements to one-line return

**Pull request merged**